vignettes/chapter13_CircadianRhythm.Rmd

```diff
@@ -0,0 +1,113 @@
+by [Rowlands et al.](https://doi.org/10.1186/s40798-019-0225-9) which looks at
+accumulated most active time which may not always be continuous in time.
+The MX metrics by Rowlands et all are discussed further down.
+
+## Cosinor analysis and Extended Cosinor analysis
+
+The (Extended) Cosinor analysis quantifies the circadian 24 hour cycle.
+To do this GGIR uses R package
+[ActCR](https://CRAN.R-project.org/package=ActCR) as a dependency.
+Specify argument `cosinor = TRUE` to perform these analysis.
+
+The implementation within GGIR part 2 is as follows:
+
+-   Acceleration values are averaged per minute, and then log
+    transformation as `log(acceleration converted to _mg_ + 1)`.
+-   Invalid data points such as caused by non-wear are set to missing
+    (NA) in order to prevent the imputation approach used elsewhere in
+    GGIR to influence the Cosinor analysis. We do this because
+    imputation technique generally come with some assumptions about
+    circadian rhythm.
+-   GGIR looks for the first valid data point in the recording and then
+    selects the maximum integer number of recording days following this
+    data point and feeds these to the ActCosinor and ActExtendCosinor
+    functions of ActCR. The time offset between the start and the
+    following midnight is then used to reverse offset the ActCR results,
+    to ensure that acrophase and acrotime can be interpreted relative to
+    midnight.
+-   In relation to Day Saving Time: Duplicated time stamps when clock
+    moves backward are ignored and missing time stamps when clock moves
+    forward are inserted as missing values.
+-   Time series corresponding to the fitted models are stored inside the
+    part 2 milestone data to facilitate visual inspection. For the
+    moment they are not used in any GGIR visualisation, but you may want
+    to look them up and try to plot them yourself.
+
+## Intradaily Variability (IV) and Interdaily Stability (IS)
+
+
+**Disclaimer: This functionality has undergone major revisions and enhancements in 2024. The documentation you find here is outdated and will be updated as soon as the enhancements have been incorporated in a GGIR release.**
+
+### IV and IS - Default
+
+The original implementation (argument IVIS.activity.metric = 1) uses the
+continuous numeric acceleration values. However, as we later realised
+this is not compatible with the original approach by van Someren and
+colleagues, which uses a binary distinction between active and inactive.
+Therefore, a second option was added (argument IVIS.activity.metric = 2),
+which needs to be used in combination with accelerometer metric ENMO,
+and collapses the acceleration values into a binary score of rest versus
+active. This is the current default.
+
+### IV and IS - Cosinor analysis compatible
+
+Disclaimer: The following has been implemented 2022, but is currently 
+undergoing critical evaluation. As a result, we may update this algorithm 
+during the course of 2023.
```

Comment:
Should this be "during the course of 2024"?

Reply:
Thanks, I have now removed the disclaimer, because there is a more general disclaimer at the start of the section on IV and IS.